app/controllers/projects_controller.rb

```diff
@@ -3,7 +3,9 @@ class ProjectsController < ApplicationController
 
   def index
     @organization_id = projects_params[:organization_id]
-    @projects = Project.where(projects_params).sort_by do |project|
+    @projects = Project.where(projects_params)
+    @projects = @projects.where('tags ?| array[:tags]', tags: params[:tags]) if params[:tags]&.any?
```

Comment:
❤️ Postgres

Reply:
Yeah this was a little too easy... I suspect problems may turn up when I try it in environments with more than my one test project.